### PR TITLE
Initialise new bom line (2.277)

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]
+1. I would suggest merging it early so that all tables-to-divs changes flow in

**Comment by timja** [comment]
> +1. I would suggest merging it early so that all tables-to-divs changes flow in

yes we did this last time too, once selection is complete assuming green will merge

**Comment by timja** [comment]
nice only token macro failing

**Comment by jglick** [comment]
> so that all tables-to-divs changes flow in

I doubt tables-to-divs regressions would routinely show up in PCT. Maybe for `configRoundtrip` tests.

File: failFast [hunk 1]
```diff
@@ -1 +1 @@
-true
+false
```
Comment (jglick) [inline failFast:1]: reminder to revert

**Comment by timja** [comment]
so this PR (merged 5 months ago 🤯) https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/402, needs releasing to fix declarative pipeline PCT

I guess we can just ignore the tests and move on?

**Review by jglick — APPROVED** [review]
Looks good, assuming this baseline does get selected.

**Comment by bitwiseman** [comment]
@timja 
https://github.com/jenkinsci/pipeline-model-definition-plugin/releases/tag/pipeline-model-definition-1.8.0

File: pct.sh [hunk 2]
```diff
@@ -114,4 +114,7 @@ rm -fv pct-work/workflow-basic-steps/target/surefire-reports/TEST-org.jenkinsci.
 # TODO until dropping 2.235.x so can rely on https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/120
 rm -fv pct-work/workflow-basic-steps/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.steps.TimeoutStepTest.xml
 
+# TODO needs release https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/402
+rm -fv pct-work/pipeline-model-definition/pipeline-model-definition/target/surefire-reports/TEST-org.jenkinsci.plugins.pipeline.modeldefinition.StageInputTest.xml
+
```
Comment (bitwiseman) [inline pct.sh:119]:
```suggestion
```

**Review by jglick — APPROVED** [review]

**Comment by timja** [comment]
auto merge has been flakey today, we've been using it a bit at work and had to click disable on it to merge some changes

**Comment by jglick** [comment]
I have also seen problems. I think it fails when you merge in new base branch changes?

**Comment by jglick** [comment]
I filed https://support.github.com/ticket/personal/0/1006996 for it (link will not work for you).

**Comment by jglick** [comment]
@timja I see you were able to cut https://github.com/jenkinsci/bom/releases/tag/bom-22 yourself. Do you prefer to release this yourself as well, or that I do so?

When I get some time I would like to switch this repo to use JEP-229, though I think without the automatic deployment (to avoid too many dependency notifications).

**Comment by timja** [comment]
Either is fine. Just haven’t had a chance, if you have time then go for it

**Comment by jglick** [comment]
blocked on https://github.com/jenkins-infra/repository-permissions-updater/pull/1817

**Comment by timja** [comment]
oh yeah, forgot about that oops 😂

**Comment by jglick** [comment]
Hmm, still getting a 403. Do I just need to wait a while?

**Comment by jglick** [comment]
```
Authorization failed for https://repo.jenkins-ci.org/releases/io/jenkins/tools/bom/bom-2.277.x/23/bom-2.277.x-23.pom 403 Forbidden
```

**Comment by timja** [comment]
requires the job to run on trusted ci

**Comment by timja** [comment]
build finished 2 minutes ago

**Comment by jglick** [comment]
There we go. https://github.com/jenkinsci/bom/releases/tag/bom-23

**Pull request merged**